app/views/angular/_player.html.erb

```diff
@@ -7,4 +7,6 @@
     {{ episodeName() }}
   </span>
 </div>
+<a ng-click="autoPlay()">{{streamExists() ? "Switch to Live Stream" : "Play Random Episode"}}</a>
```

Comment:
@omnikrll This isn't styled, but if you or @binaricorn can jump in and make this look nice, this mostly works.